nepalingo-web/src/pages/Home/Home.tsx

```diff
@@ -3,18 +3,28 @@ import { Link } from "react-router-dom";
 import logo from "../../assets/logo.png";
 import ReactGA from 'react-ga4';
 import { useAuth } from "../../components/userAuth/AuthContext";
+import GreetingCard from "../../components/GreetingCard";
 
 const Home: React.FC = () => {
-  ReactGA.send({ hitType: "pageview", page: window.location.pathname, title: "home"});
+  ReactGA.send({ hitType: "pageview", page: window.location.pathname, title: "home" });
   const { user } = useAuth();
 
+  const userName = 'user';
```

Comment:
this will just set the userName to say "Good Morning user" - see line 20 for how to get the username